testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -123,6 +146,17 @@ class MiniAppDisplayActivity : BaseActivity() {
         miniappPermissionCallback.invoke(isGranted)
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (requestCode == externalWebViewReqCode && resultCode == Activity.RESULT_OK) {
+            data?.let { intent ->
+                if (intent.hasExtra(MiniAppExternalUrlLoader.returnUrlTag))
```

Comment:
It seems kind of strange that they have to check for the `returnUrlTag` even though this intent extra is set by the SDK, so they wouldn't know that it has been set (unless we tell them). Is there any problem with having them return the `intent` directly to `emitResult(intent)`?